packages/ocaml-planet/ocaml-planet.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "sk826@cl.cam.ac.uk"
+authors: "KC Sivaramakrishnan"
+homepage: "https://github.com/kayceesrk/ocaml-planet"
+bug-reports: "https://github.com/kayceesrk/ocaml-planet/issues"
+license: "ISC"
+dev-repo: "https://github.com/kayceesrk/ocaml-planet/irmin.git"
```

Comment:
No need to resubmit.  You can just add changesets to your branch and they'll get built.  When it's all green, you can rebase the branch if desired.